contracts/MultiSigWallet.sol

```diff
@@ -200,6 +185,7 @@ contract MultiSigWallet {
         public
         returns (uint transactionId)
     {
+        require(destination == tokenContract || destination == address(this));
```

Comment:
this is a little confusing.  you should probably add a comment here on why you doing these restrictions.